src/Core/Prefab/GlobeView.js

```diff
@@ -198,7 +198,8 @@ function GlobeView(viewerDiv, coordCarto, options) {
     this.atmosphere.traverse((obj) => { obj.layers.set(atmosphereLayer); });
     this.camera.camera3D.layers.enable(atmosphereLayer);
 
-    this.scene.add(this.atmosphere);
+    wgs84TileLayer.object.add(this.atmosphere);
```

Comment:
And I'm a bit reluctant to transform `GeometryLayer` in a `THREE.Group` for now.
No particular reason though, so we might revisit this decision later.